docs/en/power_module/ark_12s_payload_power_module.md

```diff
@@ -0,0 +1,59 @@
+# ARK 12S Payload Power Module
```

Comment:
How does this differ from https://docs.px4.io/main/en/power_module/ark_12s_pab_power_module

Reply:
Specifically I mean the names are a little similar and readers might not differentiate their purpose.

Reply:
@patrikpordi Absolutely worth having separately. My point is that the reason for this  "connector to power additional devices" is not obvious, so it isn't clear why you would chose this. 

I've added a clarification in https://github.com/PX4/PX4-Autopilot/pull/25991/commits/b1a3039af5e7b5d27c324554a1b27d5f2bc0bc6b 

> This is similar to the [ARK 12S PAB Power Module](../power_module/ark_12s_pab_power_module.md) except that the additional 12V 6A supply allows easier powering of a payload.